neuralogic/core/constructs/rule.py

```diff
@@ -86,12 +90,12 @@ def __init__(self, head, body):
                     self.body[atom_index] = Relation.special.alldiff(*new_terms)
 
     def is_ellipsis_templated(self) -> bool:
-        for body_atom in self.body:
-            if not body_atom.predicate.special or body_atom.predicate.name != "alldiff":
-                continue
-            for term in body_atom.terms:
-                if term is Ellipsis:
-                    return True
+        # for body_atom in self.body:
+        #     if not body_atom.predicate.special or body_atom.predicate.name != "alldiff":
+        #         continue
+        #     for term in body_atom.terms:
+        #         if term is Ellipsis:
+        #             return True
```

Comment:
This should work but will cause an error once we replace the templated body literal